lib/fork.js

```diff
@@ -134,14 +134,16 @@ module.exports = function (file, opts) {
 		isReady = true;
 	});
 
-	promise.run = function () {
+	promise.run = function (options) {
+		options = options || {};
```

Comment:
Given that this is an internal method we should assume it's always called with an `options` object. You'll have to update `test/fork.js` though. Maybe some other places too.